Makefile

```diff
@@ -26,4 +26,12 @@ docker-image:
 	docker build -t $(IMAGE) .
 
 docker-start:
-	docker run --rm -it -v $(CURDIR)/content:/librarium/content/ -p 9000:9000 $(IMAGE)
\ No newline at end of file
+	docker run --rm -it -v $(CURDIR)/content:/librarium/content/ -p 9000:9000 $(IMAGE)
+
+verify-url-links:
+	rm link_report.csv || echo "No report exists. Proceeding to scan step"
+	linkinator https://docs.spectrocloud.com/ --recurse --timeout 60000 --format csv >> link_report.csv
```

Comment:
I think this should this run against the existing PR, more or less of the safety check in your PR. It could be a GitHub action that runs and comments if there are new issues.